nuget.config

```diff
@@ -1,18 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <config>
```

Comment:
This key only applies to when using a *package.config* according to the docs: https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#config-section